tests/dragonfly/search_test.py

```diff
@@ -479,7 +479,9 @@ def test_redis_om(df_server):
     try:
         import redis_om
     except ModuleNotFoundError:
-        pytest.skip("Redis-om not installed")
+        if os.getenv("GITHUB_ACTIONS") == None:
+            pytest.skip("Redis-om not installed")
```

Comment:
Why don't we just install those as part of our workflow files and then the tests can run without skip?

Reply:
ohhh, right the `os.getenv("GITHUB_ACTIONS") == None` means if we are not on `GH ACTIONS machine`. nice!